src/main/java/de/bayern/gdi/services/WFSMetaExtractor.java

```diff
@@ -96,7 +96,9 @@
 
     private static final String XPATH_OUT_FORMATS
         = "/wfs:WFS_Capabilities"
-        + "/ows:OperationsMetadata/ows:Parameter[@name='outputFormat']"
+        + "/ows:OperationsMetadata/"
+        + "/ows:Operation[@name='GetFeature']"
+        + "/ows:Parameter[@name='outputFormat']"
         + "/ows:AllowedValues/ows:Value/text()";
 
```

Comment:
I think we have to do both. We have per operation output formats _and_ top level output formats. Same logic as with the paging stuff.